### PR TITLE
Multiple node balances and details

### PR DESCRIPTION
**Pull request by mjadach-iv**
Added balance in the table.
'Show details' button switches the top of the page to the selected node
image

### CODE REVIEW

**Review by Jaguaras — CHANGES_REQUESTED** [review]
Please leave 2 decimals, it should be enough. Because small screens ( with resolution 3024 × 1964) can't see much.

File: src/store/slices/stakingHub/actionsAsync.ts [hunk 1]
```diff
@@ -201,10 +201,11 @@ const getSubgraphDataThunk = createAsyncThunk<
       if (json.balances.length > 0) output.overall_staking_v2_balances = json.balances[0];
 
       console.log('output.registeredNodesInNetworkRegistry', output.registeredNodesInNetworkRegistry)
-      if (output.registeredNodesInNetworkRegistry?.length > 0) {
-        let nodeAddress = output.registeredNodesInNetworkRegistry[0].node.id;
-        console.log('nodeAddress', nodeAddress)
-        dispatch(getNodeDataThunk(nodeAddress));
+      if (output.registeredNodesInSafeRegistry?.length > 0) {
```
Comment (esterlus) [inline src/store/slices/stakingHub/actionsAsync.ts:204]:
```  
    output.registeredNodesInSafeRegistry?.forEach(safeRegNode => {
          let nodeAddress = safeRegNode.node.id;
          dispatch(getNodeDataThunk({nodeAddress, browserClient}));
        })
```

**Review by esterlus — APPROVED** [review]
found a code optimization with forEach

File: src/store/slices/stakingHub/actionsAsync.ts (hunk 1 above)
Comment (mjadach-iv) [reply src/store/slices/stakingHub/actionsAsync.ts:204]: Updated

**Review by Jaguaras — APPROVED** [review]
lgtm!

**Pull request merged**